assets/runtime/functions

```diff
@@ -232,6 +233,17 @@ gitlab_configure_redis() {
     REDIS_DB_NUMBER
 }
 
+gitlab_configure_actioncable() {
+  echo -n "Configuring gitlab::actioncable"
+
+  gitlab_finalize_redis_parameters
+  gitlab_check_redis_connection
+
+  update_template ${GITLAB_ACTIONCABLE_CONFIG} \
+    REDIS_HOST \
+    REDIS_PORT
```

Comment:
I think you need to add REDIS_DB_NUMBER here

Reply:
nice catch, added

Reply:
Nice work!